keyboards/s7_elephant/keymaps/default/keymap.c

```diff
@@ -0,0 +1,95 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+enum custom_layers {
+    _QWERTY,
+    _FN
+};
+
+enum custom_keycodes {
+    EPRM = SAFE_RANGE,
```

Comment:
This can be removed as there is officially an EEPROM reset keycode now.

https://docs.qmk.fm/#/quantum_keycodes?id=qmk-keycodes

`EEPROM_RESET` and `EEP_RST`

Reply:
Totally missed that keycode. Fixed.